### PR TITLE
Update it, nl type and busway translations

### LINKED ISSUE #5623: Support translations for highway=busway and bridge=yes
Actually translations is not present for highway=busway and bridge=yes.

https://overpass-turbo.eu/s/1yaU

### LINKED ISSUE #5688: Some Dutch and Italian translations
As a follow-up of some translations I added quite a while ago, I'm uploading a few additions to types_strings.txt here. I'm still not technical enough to figure out how exactly I'm supposed to present this in a PR or whatever, so it's just the raw text file. 

The changes consist mainly of new Dutch and Italian translations, and a little bit of English, e.g. "culvert" for waterway.ditch.tunnel.

[types_strings.txt](https://github.com/organicmaps/organicmaps/files/12307456/types_strings.txt)

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
Fixes #5688
Fixes #5623

### CODE REVIEW

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -432,7 +434,7 @@
     zh-Hant = 滑行道
 
   [type.aeroway.terminal]
-    en = Terminal
+    en = Airport terminal
```
Comment (Jean-BaptisteC) [inline data/strings/types_strings.txt:437]: Is it necessary ?

**Review by Jean-BaptisteC — COMMENTED** [review]
I suggest to remove identical translations as English?

File: data/strings/types_strings.txt (hunk 1 above)
Comment (biodranik) [inline data/strings/types_strings.txt:437]:
```suggestion
    en = Terminal
```

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -27694,6 +27713,8 @@
 
   [type.waterway.ditch.tunnel]
     ref = type.waterway.ditch
+    en = Culvert
```
Comment (biodranik) [inline data/strings/types_strings.txt:27716]: Good! Can you please add the tunnel translations for other languages too using [Wiki](https://en.wikipedia.org/wiki/Culvert) and other translation services?

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -27733,6 +27754,8 @@
 
   [type.waterway.drain.tunnel]
     ref = type.waterway.drain
+    en = Culvert
+    nl = Duiker
```
Comment (biodranik) [inline data/strings/types_strings.txt:27758]:
```suggestion
    ref = type.waterway.ditch.tunnel
```

File: data/strings/types_strings.txt (hunk 2 above)
Comment (Jean-BaptisteC) [reply data/strings/types_strings.txt:27716]: Done,
Some languages are not available in Wikipedia, i have check translation on DeepL but satisfying.
I wait translations are approved to regenerate strings

**Comment by biodranik** [comment]
@501Ghost these other terminals are mostly not present in Organic Maps or are called differently.

File: data/strings/types_strings.txt [hunk 4]
```diff
@@ -7128,6 +7134,8 @@
     es = Té de burbujas
     et = Mullitee
     eu = Burbuila tea
+    it = Bubble tea
+    nl = Bubbelthee
```
Comment (jimcarst) [inline data/strings/types_strings.txt:7138]:
```suggestion
    nl = Bubble tea
```


Nobody would call it Bubbelthee so sticking with English is better.

**Review by jimcarst — COMMENTED** [review]
Left one suggestion. The rest is all ok for Nl.

**Comment by Jean-BaptisteC** [comment]
@biodranik @vng PTAL

**Review by biodranik — APPROVED** [review]
Thank you!

**Pull request merged**